camply/search/search_going_to_camp.py

```diff
@@ -175,7 +175,7 @@ def get_all_campsites(self) -> List[AvailableCampsite]:
                     # be viable for camping. Skip all zero-capacity sites.
                     if (
                         not site_details["minCapacity"]
-                        or not site_details["maxCapacity"]
+                        and not site_details["maxCapacity"]
```

Comment:
Here's where we allow sites with a maxCapacity, but not a minCapacity, to get selected